modules/SofaGuiQt/src/sofa/gui/qt/RealGUI.cpp

```diff
@@ -429,7 +429,7 @@ RealGUI::RealGUI ( const char* viewername)
     fpsLabel->setMinimumSize ( fpsLabel->sizeHint() );
     fpsLabel->clear();
 
-    timeLabel = new QLabel ( "Time: 999.9999 s", statusBar() );
+    timeLabel = new QLabel ( "Time: 999.9999", statusBar() );
```

Comment:
Everywhere in SOFA Time and Timestep are without unit?
I understand but I find weird/confusing to have a value shown without unit. Maybe we should also add the number of iteration side to side. Or another info to tell user that Time shown ==  dt * nbstep.  (except if dt has been modified during the simu ...)

Reply:
I agree that time is not necessarily in seconds. Simulations in geophysics have a dt in thousands of years. It's the same of others physical values: they don't have units, as long as all of them are consistent.
I also agree to show the number of iterations. I already needed it